scio-test/src/test/scala/com/spotify/scio/values/SCollectionTest.scala

```diff
@@ -497,7 +497,7 @@ class SCollectionTest extends PipelineSpec {
         r should containInAnyOrder (Seq(1, 2, 3))
       }
     }
-    stdOutMock.message should contain theSameElementsAs Seq("1", "2", "3")
+    stdOutMock.message should contain allElementsOf Seq("1", "2", "3")
```

Comment:
`theSameElementsAs` has to contain all and **only** the elements of expected, while `allElementsOf` all the elements of expected.